Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -329,6 +329,24 @@ void UPSWorldSubsystem::OnTakeActorsFromPoolCompleted(const TArray<FPoolObjectDa
 	}
 }
 
+// Returns spot component by the player tag, returns null if spot is not found
+UPSSpotComponent* UPSWorldSubsystem::GetSpotComponentByPlayerTag(FPlayerTag PlayerTag)
+{
+	if (PSSpotComponentArrayInternal.IsEmpty())
```

Comment:
please also do early return if player tag is not valid `!PlayerTag.IsValid()`

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/40/commits/b63ec57a0fa35678e1367c7c433d3290fa6c1b22